test/gurka/test_traffic.cc

```diff
@@ -187,23 +187,25 @@ TEST(Traffic, CutGeoms) {
 
   // then we add one portion of the edge having traffic
   {
-    valhalla::baldr::TrafficSpeed ts{valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
-                                     valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
-                                     valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
-                                     valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
-                                     0u,
-                                     0u,
-                                     0u,
-                                     0u,
-                                     0u};
+    valhalla::baldr::TrafficSpeed ts{
+        valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
+        valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
+        valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
+        valhalla::baldr::UNKNOWN_TRAFFIC_SPEED_RAW,
+        0u,
+        0u,
+        0u,
+        0u,
+        0u,
+        0u,
```

Comment:
used one of the bits that was spare to say an edge has an incident on it (or is expected to, not gauranteed)